modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm

```diff
@@ -9,35 +9,35 @@
 
 /datum/armament_entry/company_import/deforest/first_aid_kit/civil_defense
 	item_type = /obj/item/storage/medkit/civil_defense/stocked
-	cost = PAYCHECK_COMMAND * 2.5
+	cost = PAYCHECK_COMMAND * 5 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```

Comment:
/datum/supply_pack/medical/civil_defense needs an adjustment to
cost = CARGO_CRATE_VALUE * 10

Reply:
Already is